sem/utils.py

```diff
@@ -713,25 +724,32 @@ def filter_logs(db,
     """
     query_final = []
 
-    if sevirity_class is not None or components is not None:
-        if isinstance(sevirity_class, str):
-            sevirity_class = [sevirity_class]
-
+    if severity_class is not None or components is not None:
```

Comment:
components was check but never used. is there a reason for this?

Reply:
Refer [65afd95](https://github.com/signetlabdei/sem/pull/51/commits/65afd95372eaa48713eea1d6158b3b3d7d9306a9).